boards/nxp/frdm_mcxe31b/frdm_mcxe31b.dts

```diff
@@ -159,6 +159,10 @@
 	overdrive = <12>;
 };
 
+&pit_0 {
```

Comment:
As pit is an optional counter, is it necessary to enable it on board level without any channel enabled?

Reply:
The board dts guide has been updated since I last viewed, need a confirm https://github.com/zephyrproject-rtos/zephyr/pull/99759#discussion_r2622499535